Modules/Filtering/ImageFilterBase/include/itkRecursiveSeparableImageFilter.h

```diff
@@ -253,7 +254,7 @@ class ITK_TEMPLATE_EXPORT RecursiveSeparableImageFilter : public InPlaceImageFil
 private:
   /** Direction in which the filter is to be applied
    * this should be in the range [0,ImageDimension-1]. */
-  unsigned int m_Direction{ 0 };
```

Comment:
We need to keep the semicolon.

Reply:
Yeah, I thought that didn't seem right, but KWStyle was requesting it. It turns out this was a problem with an outdated KWStyle in /usr/bin on my system. After building the version distributed with ITK instead, it is no longer requiring this change.